phpstan.neon

```diff
@@ -1,138 +1,25 @@
 includes:
 	- vendor/larastan/larastan/extension.neon
 	- vendor/lychee-org/phpstan-lychee/phpstan.neon
-
+	- phpstan-baseline.neon
 parameters:
+	level: 1
 	treatPhpDocTypesAsCertain: false
 	paths:
 		- app
-		- scripts
+		- config
 		- lang
 		- database/migrations
-		- tests
-		- config
+		- scripts
 	excludePaths:
-		- database/migrations/2021_12_04_181200_refactor_models.php
 		- app/Services/Archives # This is creating errors depending if version 2.1 or 3.1 is installed...
 	stubFiles:
 		# these can be removed after https://github.com/thecodingmachine/safe/issues/283 has been merged
 		- phpstan/stubs/Wireable.stub
 		- phpstan/stubs/image.stub
 		- phpstan/stubs/imageexception.stub
 	ignoreErrors:
-		# False positive php8.2 + phpstan on interface @property
```

Comment:
I strongly suggest that you keep those but just comment them. They might come handy later when you increase the levels.
Also that is very likely going to be breaking the script/delete_legacy.sh

Reply:
I mean, we can always come back, git should help us then

Reply:
Sure.